messages/commonFlags.md

```diff
@@ -0,0 +1,53 @@
+# promote.devops-center-project-name.summary
+
+Name of the DevOps Center project.
+
+# promote.branch-name.summary
+
+Name of the branch from which to deploy changes to the stage’s org.
+
+# promote.deploy-all.summary
+
+Deploy all metadata in branch.
+
+# promote.deploy-all.description
+
+If you don’t specify this flag, only changes in this stage’s branch are deployed.
+
+# promote.devops-center-username.summary
+
+Username or alias for the DevOps Center org.
+
+# promote.bundle-version-name.summary
+
+Version name of the bundle.
+
+# promote.bundle-version-name.description
+
+Bundle version name when deploying to the first stage after the bundling stage.
```

Comment:
Found this in the docs: "You must indicate the bundle version if deploying to the environment that correspond to the first stage after the bundling stage."

Reply:
Fixed